tests/unit/fem/test_build_dof_to_arrays.cpp

```diff
@@ -225,15 +260,35 @@ TEST_CASE("Build Dof To Arrays (Parallel)",
 
                if (l < 0 || l >= dofs.Size()) { num_rang_fails++; }
 
-               int ldof = (dofs[l] >= 0) ? dofs[l] : (-1 - dofs[l]);
+               int elemdof = (dofs[l] >= 0) ? dofs[l] : (-1 - dofs[l]);
+
+               if (i != elemdof) { num_ldof_fails++; }
+
+               int bdr_e = fespace.GetBdrElementForDof(i);
+               int bdr_l = fespace.GetBdrLocalDofForDof(i);
+
+               if (all_bdr_ldofs.Find(i) >= 0) // if this is a bdr ldof
+               {
+                  if (bdr_e < 0 || bdr_e >= pmesh.GetNBE()) { num_bdr_elem_fails++; }
```

Comment:
Would be better to add `continue` here and elsewhere to not get into segfaults if it fails.